app/views/layouts/application.html.erb

```diff
@@ -12,4 +12,8 @@
   <body>
```

Comment:
Add please more useful interface for user, just main page( root_url when you open your app) with all links and ability to move to all of your pages from there and back, because now it is not user friendly.